### DISCUSSION TITLE
Can't install xmpp.js using NPM

### DISCUSSION THREAD

**Discussion by zach-little** [xmppjs/xmpp.js#981] [General]
NPM Version: 9.5.1
Node Version: 19.8.1

I am running `npm install xmpp.js` expecting to get the latest 0.13.1 version, however it keeps giving me an error saying No matching version found for @xmpp/debug@^0.13.1. I have started my project over and to no avail have I been able to install the latest version of xmpp.js. Anyone have any thoughts on how to get this to install correctly?

![image](https://user-images.githubusercontent.com/57122612/229716239-6754fc57-90a2-4746-af03-8a0d89f40799.png)

**Comment by sonnyp** [comment]
Install `@xmpp/client`

**Reply by zach-little** [reply]
I'd like to install the entire xmpp.js module. What I am running into is that @xmpp/test in its package.json has a dependency for @xmpp/debug for version 0.13.1 which doesn't exist. Can this be updated to point to the correct version?

Location of bugged @xmpp/test package.json file: https://registry.npmjs.org/@xmpp/test/-/test-0.13.0.tgz (Which is where it pulls from when I run npm i xmpp.js)

![image](https://user-images.githubusercontent.com/57122612/230644422-1f3777d1-5120-4da7-9e2b-eb482ff3913e.png)

**Comment by zach-little** [comment]
I already installed @xmpp/client and @xmpp/debug (version 0.13.1 and 0.13.0 respectively), however, that has slightly different syntax for setup than if I had the xmpp.js package correct? I have a solution already setup for version 0.12.1 that failed to work after not being used a couple years, came back and realized that 0.13.1 now existed so I was just trying to upgrade to that.
